src/containers/FrontpageEditPage/FrontpageEditPage.tsx

```diff
@@ -169,30 +173,32 @@ const RootFields = () => {
     <FrontpageArticleWrapper>
       <Heading textStyle="title.large">{t("htmlTitles.editFrontpage")}</Heading>
       <Wrapper>
-        <ArticleTitle>
-          <span>
-            {articleField.value ? (
-              <>
-                {t("frontpageForm.frontpageArticle")}
-                <StyledSafeLink to={toEditFrontPageArticle(articleField.value.id, i18n.language)} target="_blank">
-                  {articleField.value.title.title}
-                </StyledSafeLink>
-              </>
-            ) : (
-              t("frontpageForm.noFrontpageArticle")
-            )}
-          </span>
+        <EditFrontpageWrapper>
+          {articleField.value ? (
+            <>
+              <Text>{t("frontpageForm.frontpageArticle")}</Text>
+              <SafeLink
+                to={toEditFrontPageArticle(articleField.value.id, i18n.language)}
+                target="_blank"
+                css={linkRecipe.raw()}
+              >
+                {articleField.value.title.title}
+              </SafeLink>
```

Comment:
Burde ikke denne ha default lenke-styling?

Reply:
Var litt i tvil, gikk for styling vi bruker en del andre steder i ed, men usikker?

Reply:
Det er kanskje fra gammelt av?

Reply:
Egentlig ikke, men er akkurat det samme for meg, så vi kan godt la den være default

Reply:
La oss prøve på default!